test/pkg.jl

```diff
@@ -100,6 +100,8 @@ temp_pkg_dir() do
     str = chomp(takebuf_string(iob))
     @test endswith(str, string(Pkg.installed("Example")))
     @test isempty(Pkg.dependents("Example"))
+    # Checking to see dependency reading
+    @test Pkg.dependencies("Example") == Dict("julia" => Pkg.Types.VersionSet(v"0.3.0", typemax(VersionNumber)))
```

Comment:
ComputeFramework isn't registered right? So that information isn't present in metadata. It should be consistent about always being a metadata query or always being an installed packages query, depending on what the use case is. There is no use of these in base right now.